tests/NonPhpFile/NetteDINeonMethodCallRenamer/NetteDINeonMethodCallRenamerTest.php

```diff
@@ -0,0 +1,148 @@
+            new MethodCallRename(FirstService::class, 'add', 'addConfig')
+        );
+
+        $file = new File($fixtureFileInfo, $fixtureFileInfo->getContents());
+        $this->netteDINeonMethodCallRenamer->process([$file]);
+
+        $expected = "services:
+    firstService:
+        factory: Rector\Core\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\FirstService
+        setup:
+            - addConfig('key1', 'value1')
+            - addConfig('key2', 'value2')
+
+    -
+        class: Rector\Core\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\SecondService('service-name')
+        setup:
+            - add('first-key', 'first-value')
+            - add('second-key', 'second-value')
+";
+
+        $this->assertTrue($file->hasChanged());
+        $this->assertEquals($expected, $file->getFileContent());
+    }
+
+    /**
+     * @dataProvider provideData()
+     */
+    public function testRenameAddToAddConfigForSecondService(SmartFileInfo $fixtureFileInfo): void
+    {
+        $this->methodCallRenameCollector->addMethodCallRename(
+            new MethodCallRename(SecondService::class, 'add', 'addConfig')
+        );
+
+        $file = new File($fixtureFileInfo, $fixtureFileInfo->getContents());
+        $this->netteDINeonMethodCallRenamer->process([$file]);
+
+        $expected = "services:
+    firstService:
+        factory: Rector\Core\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\FirstService
+        setup:
+            - add('key1', 'value1')
+            - add('key2', 'value2')
+
+    -
+        class: Rector\Core\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\SecondService('service-name')
+        setup:
+            - addConfig('first-key', 'first-value')
+            - addConfig('second-key', 'second-value')
+";
+
+        $this->assertTrue($file->hasChanged());
+        $this->assertEquals($expected, $file->getFileContent());
+    }
+
+    /**
+     * @dataProvider provideData()
+     */
+    public function testRenameAddToAddConfigForServiceInterface(SmartFileInfo $fixtureFileInfo): void
+    {
+        $this->markTestSkipped('Skipped - we need to load all classes via type');
+
+        $this->methodCallRenameCollector->addMethodCallRename(
+            new MethodCallRename(ServiceInterface::class, 'add', 'addConfig')
+        );
+
+        $file = new File($fixtureFileInfo, $fixtureFileInfo->getContents());
+        $this->netteDINeonMethodCallRenamer->process([$file]);
+
+        $expected = "services:
+    firstService:
+        factory: Rector\Core\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\FirstService
+        setup:
+            - addConfig('key1', 'value1')
+            - addConfig('key2', 'value2')
+
+    -
+        class: Rector\Core\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\SecondService('service-name')
+        setup:
+            - addConfig('first-key', 'first-value')
+            - addConfig('second-key', 'second-value')
+";
```

Comment:
This should be decopuled in fixture-style as other tests.

```
a
-----
b
```

so its reasier to extend.